src/algorithms.jl

```diff
@@ -252,6 +252,28 @@ Base.@pure QNDF1(;chunk_size=0,autodiff=true,diff_type=Val{:central},
 
 Base.@pure QBDF1(;kwargs...) = QNDF1(;kappa=0,kwargs...)
 
+struct QNDF2{CS,AD,F,FDT,K,T,T2,κType,Controller} <: OrdinaryDiffEqNewtonAdaptiveAlgorithm{CS,AD,Controller}
+  linsolve::F
+  diff_type::FDT
+  κ::K
+  tol::T
+  extrapolant::Symbol
+  min_newton_iter::Int
+  max_newton_iter::Int
+  new_jac_conv_bound::T2
+  kappa::κType
+end
+Base.@pure QNDF2(;chunk_size=0,autodiff=true,diff_type=Val{:central},
+                      linsolve=DEFAULT_LINSOLVE,κ=nothing,tol=nothing,
+                      extrapolant=:linear,min_newton_iter=1,
+                      max_newton_iter=7,new_jac_conv_bound = 1e-3,kappa = -1//9,
```

Comment:
Shouldn't this be a tuple of two kappas?

Reply:
why two values? as given in the paper, for 2nd order `qndf` `kappa` is `-1/9`.